### PR TITLE
My Friends Page

### PR DESCRIPTION
**Pull request by adiep2**

### CODE REVIEW

**Review by ccchoi2 — APPROVED** [review]
LGTM! 🚀

File: CMPUT_301_Project/app/src/main/res/layout/activity_add_friends.xml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        android:text="Pending Requests"
+        app:layout_constraintBottom_toTopOf="@+id/scrollView"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.498"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/linearLayout"
+        app:layout_constraintVertical_bias="0.355"
+        android:textSize="32sp" />
+
+    <androidx.fragment.app.FragmentContainerView
```
Comment (Matheus-Du) [inline CMPUT_301_Project/app/src/main/res/layout/activity_add_friends.xml:85]: Navbar comes up with keyboard to block some of the screen. Not an issue per se but might look better to not have this. I'll leave it up to you.

image

File: CMPUT_301_Project/app/src/main/res/layout/activity_add_friends.xml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            android:layout_height="40dp"
+            android:layout_marginRight="5dp"
+            android:layout_gravity="center"
+            android:text="\u2713"
+            android:background="@drawable/circle_green" />
+
+    </LinearLayout>
+
+    <TextView
+        android:id="@+id/textView"
```
Comment (Matheus-Du) [inline CMPUT_301_Project/app/src/main/res/layout/activity_add_friends.xml:73]: Can either remove id or make it more specific

File: CMPUT_301_Project/app/src/main/res/layout/activity_my_friends.xml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+            android:layout_height="match_parent"
+            android:nestedScrollingEnabled="true"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent">
+        </ListView>
+
+    </androidx.core.widget.NestedScrollView>
+
+    <TextView
+        android:id="@+id/textView"
```
Comment (Matheus-Du) [inline CMPUT_301_Project/app/src/main/res/layout/activity_my_friends.xml:34]: Can probably remove this id or make it more specific

File: CMPUT_301_Project/app/src/main/res/layout/activity_my_friends.xml [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+        <TextView
+            android:id="@+id/deleteFriendText"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_margin="6dp"
+            android:layout_weight=".5"
+            android:visibility="gone"
+            android:gravity="center"
+            android:text="Click Any Habit to Delete It"
```
Comment (Matheus-Du) [inline CMPUT_301_Project/app/src/main/res/layout/activity_my_friends.xml:80]: Should prompt to remove friends instead of delete habits

**Review by Matheus-Du — CHANGES_REQUESTED** [review]
Looks good, just a couple small things to look at and one typo in a textView.

**Review by Matheus-Du — APPROVED** [review]

**Pull request merged**